flink-formats/flink-avro-confluent-registry/src/test/java/org/apache/flink/formats/avro/registry/confluent/debezium/DebeziumAvroSerDeSchemaTest.java

```diff
@@ -222,4 +225,18 @@ public void close() {
             // do nothing
         }
     }
+
+    private class MockInitializationContext
```

Comment:
Do you think this might be reusable, or is it too simple to bother?

Reply:
seems fine as is. Should be static though.